manifests/04-deployment.yaml

```diff
@@ -24,16 +24,17 @@ spec:
         - "-configmap=cluster-monitoring-config"
         - "-logtostderr=true"
         - "-v=4"
-        #- "-tags=prometheus-operator=master"
-        #- "-tags=prometheus-config-reloader=master"
-        #- "-tags=config-reloader=master"
-        #- "-tags=prometheus=master"
-        #- "-tags=alertmanager=master"
-        #- "-tags=grafana=master"
-        #- "-tags=oauth-proxy=master"
-        #- "-tags=node-exporter=master"
-        #- "-tags=kube-state-metrics=master"
-        #- "-tags=kube-rbac-proxy=master"
+        - "-images=prometheus-operator=docker.io/openshift/origin-prometheus-operator:latest"
```

Comment:
@pgier can you check that we indeed have all of these images and that we do also have a latest tag for each of them that we are pushing to?